packaging/prerelease/s3_index.sh

```diff
@@ -5,35 +5,35 @@ set -e -u -o pipefail # Fail on error
 dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 cd $dir
 
+# If editing this file, you may also want to update windows/s3_prerelease.cmd
+
 client_dir="$dir/../.."
 bucket_name=${BUCKET_NAME:-}
-save_dir="/tmp/s3index"
+platform=${PLATFORM:-}
 
 if [ "$bucket_name" = "" ]; then
   echo "No BUCKET_NAME"
   exit 1
 fi
 
+if [ "$platform" = "" ]; then
+  echo "No PLATFORM"
+  exit 1
+fi
```

Comment:
If you want, you can leave out this check entirely and just use `$PLATFORM` everywhere. The `set -u` at the top of the file will cause that to be an error if it's undefined.